setup-local/test/binaryControl.test.js

```diff
@@ -40,25 +41,33 @@ describe('Binary Control Operations', () => {
   });
 
   context('Private Methods Behaviour', () => {
+    before(() => {
+      process.env.GITHUB_WORKSPACE = '/some/work/space';
+    });
+
+    after(() => {
```

Comment:
is there a need for `after` when we have https://github.com/browserstack/github-actions/pull/4/files#diff-0eebff27d2bbe9547b592d0f65f3b406R44

Reply:
It just from readability sense that we are starting with some `workspace` and the set it to empty while leaving. That env var is not going to change any state for the user's machine anyway.

Reply:
resolving it.